e2e/LongHaul/module/Parameters.cs

```diff
@@ -20,12 +20,19 @@ public enum JsonSerializingLibrary
 
     internal class Parameters
     {
+        [Option(
+            'g',
+            "GatewayHostName",
+            Required = true,
+            HelpText = "The gateway edge device to connects to.")]
+        public string GatewayHostName { get; set; }
+
         [Option(
             'c',
             "ConnectionString",
             Required = false,
-            HelpText = "The connection string for the module to simulate.")]
-        public string ConnectionString { get; set; } = Environment.GetEnvironmentVariable("IOTHUB_LONG_HAUL_MODULE_CONNECTION_STRING");
+            HelpText = "The connection string for the edge module to simulate.")]
```

Comment:
if we still want this workable for both device module and edge module, 
```suggestion
            HelpText = "The connection string for the device/edge module to simulate.")]
```

Reply:
Looks like he's using 2 different parameters, one for each type, because the environment variables are different.